### DISCUSSION TITLE
Share your projects

### DISCUSSION THREAD

**Discussion by unocelli** [frangoteam/FUXA#390] [Show and tell]
Here we can share projects, views (json files or link) with images

**Comment by superkalt** [comment]
the first attempt with FUXA @ DOCKER on SYNOLGY. Data source is MQTT.

[https://fuxa.superkalt.de](https://fuxa.superkalt.de)

no password protection has been created yet, you can play with it and hopefully won't break anything. The temperature data is real, there is no access to the plant!

please give feedback if something doesn't work!

**Reply by superkalt** [reply]
![image](https://user-images.githubusercontent.com/44277174/173446133-3e0065e2-f7d3-45ea-bc94-ce70226b295b.png)
